CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+* Ignore some paths in CI
```

Comment:
You can revome this line from the CHANGELOG.md (developers don't care about CI)

Reply:
done